streamz/dataframe/core.py

```diff
@@ -314,7 +321,18 @@ class Series(Frame, _SeriesMixin):
     --------
     DataFrame
     """
-    _subtype = pd.Series
+
+    def __init__(self, *args, **kwargs):
+        example = None
+        if "example" in kwargs:
+            example = kwargs.get('example')
+        elif len(args) > 1:
+            example = args[1]
+        if isinstance(self, Index):
+            is_frame_like(self, is_index_like, example)
```

Comment:
I think it would be cleaner if `is_frame_like` (with a better name) passed a value back, which was then set on the instance attribute. Still would be nice to have *something* showing that a pandas-series and cu-series instance are related, could there be more than one attribute, or some inheritance?

Reply:
Made changes to the method. Not completely sure if it's close to something that you are suggesting.